docs/src/devdocs/interpolations.md

```diff
@@ -1,12 +1,51 @@
+```@meta
+Draft = false
+```
+
 # [Interpolations](@id devdocs-interpolations)
 
-## Type definitions
+All `Interpolation`s should subtype `Interpolation{shape, order}`,
+where `shape <: AbstractRefShape` is the reference shape for which
+the interpolation is defined and `order` is the characteristic interpolation
+order. The [how-to at bottom of this page](@ref devdocs-howto_new-interpolation) describes how to implement a new interpolation.
```

Comment:
Should we add a comment what we mean by characteristic order? I think it can be surprising for some users that first order Lagrange on Quad and Hex is actually non-linear. :)

Reply:
True, perhaps `where order is the highest complete polynomial degree described by the interpolation on the reference shape.`? 

Potentially add in the examples that `(E.g. a triangle with linear shape functions have degree 1, and so does the quadrilateral with bilinear shape functions.)`?

Reply:
Well, actually I think this should rather be described [here](https://ferrite-fem.github.io/Ferrite.jl/stable/reference/interpolations/) since that is public API. 

For the devdocs, I think it is reasonable to demand that users first are aquatinted with the public API / docs so that we don't have to repeat too much?

Reply:
> [...] where order is the highest complete polynomial degree described by the interpolation on the reference shape.

I think with order we usually refer to the "convergence order" of the element in "compatible problems". For example, the lowest monomial appearing in a Q1rot elements is of order 2, but the interpolation order is just 1.

Reply:
> For the devdocs, I think it is reasonable to demand that users first are aquatinted with the public API / docs so that we don't have to repeat too much?

I can just partially agree here. I think repeating information is sometimes okay. However, we could at least add a reference to the relevant part of the public API.

Reply:
Added links to the public API documentation.